design/sketch.webidl

```diff
@@ -635,23 +639,28 @@ interface WebGPUDevice {
     WebGPUCommandBuffer createCommandBuffer(WebGPUCommandBufferDescriptor descriptor);
     WebGPUFence createFence(WebGPUFenceDescriptor descriptor);
 
-    WebGPUQueue getQueue();
-
     attribute WebGPULogCallback onLog;
     WebGPUObjectStatusQuery getObjectStatus(StatusableObject statusableObject);
 };
 
+interface WebGPUQueueFamily {
+    readonly attribute boolean graphics;
```

Comment:
8 different possibilities? The [DX12 style](https://docs.microsoft.com/en-us/windows/desktop/api/d3d12/ne-d3d12-d3d12_command_list_type) is easier to work with.